app/package.json

```diff
@@ -26,8 +26,9 @@
     "react-datepicker": "^7.5.0",
     "react-dom": "^18",
     "react-hook-form": "^7.53.2",
-    "react-icons": "^5.3.0",
+    "react-icons": "^5.4.0",
```

Comment:
I just "installed" this blindly with `npm install` and it looks like I have updated it instead. It's a minor bump, so I'm going to leave it, but I do want to flag it because it's not impossible i regressed something somewhere. Feels like a safe enough bet to me though.